pkg/ddl/notifier/subscribe.go

```diff
@@ -255,7 +278,27 @@ func (n *DDLNotifier) processEventForHandler(
 	return nil
 }
 
-// Close releases the resources.
-func (n *DDLNotifier) Close() {
-	n.ownedSess.Close()
+// Stop stops the background loop.
+// Exposed for testing.
+// Do not call this function directly. Use owner.Listener interface instead.
+func (n *DDLNotifier) Stop() {
+	// If the notifier is not started, the cancel function is nil.
```

Comment:
maybe mention that in fact cancel function will not be nil. Just to improve robust.

Also maybe 

```
if n.cancel == nil { return }
n.cancel()
n.wg.Wait()
```

to reduce indent. Seems linter is broken.

Reply:
> maybe mention that in fact cancel function will not be nil. Just to improve robust.

In the test, it is possible.

Reply:
> to reduce indent. Seems linter is broken.

Updated.